tests/unit/test_mongo.py

```diff
@@ -1,34 +1,33 @@
 import os
 import pytest
-import subprocess
-import time
 from pymongo import MongoClient
+from pymongo.errors import ConnectionFailure
 from icecream import ic
-
 from src.grpc_.utils import wait_for_services
 
 @pytest.fixture(scope="session", autouse=True)
 def setup_mongo():
-    os.system(
-        "docker compose -f deploy/mongo/compose.yml up --build -d"
-    )
-    wait_for_services(["mongo", "mongo-express"], timeout=30, init_time=10)
+    os.system("docker compose -f deploy/mongo/compose.yml up --build -d")
+    wait_for_services(["mongo", "mongo-express"], timeout=10, init_time=5)
     yield
-    os.system(
-        "docker compose -f deploy/mongo/compose.yml down"
-    )
+    os.system("docker compose -f deploy/mongo/compose.yml down")
 
-@pytest.mark.slow # the setup is slow but we have to mark here
-@pytest.mark.skip("Not configured for testing quite yet")
+@pytest.mark.slow
 def test_mongo():
-    host = os.environ.get("host", "mongo")
+    host = os.environ.get("host", "localhost")
```

Comment:
the compose services make their own vLAN where everything is accessed according to service name -- so mongo is correct. localhost wouldn't be found.